js/api.js

```diff
@@ -0,0 +1,31 @@
+const Url = {
+  SERVER: ' https://23.javascript.pages.academy/kekstagram ',
+  DATA: 'https://23.javascript.pages.academy/kekstagram/data',
+};
+
+const getData = (onSuccess, onFail) => {
+  fetch(Url.DATA)
+    .then((response) => response.json())
+    .then((photos) => onSuccess(photos))
```

Comment:
```suggestion
    .then((data) => onSuccess(data))
```

это универсальная функция, поэтому абстрактные названия параметров